packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md

```diff
@@ -23,31 +23,31 @@ This page provides the API rate limits for authentication and end-user activitie
 | Action and Okta API Endpoint                                                                                           | Developer (free) | Developer (paid) | One App | Enterprise | Workforce Identity    |
 | ---------------------------------------------------------------------------------------------------------------------- | ----------------: | ----------------: | -------: | ----------: | ---------------------: |
 | **Cumulative rate limit**                                                                                              | **2,100**        | **10,800**        | **10,800**| **13,000**| **18,250**            |
-| **Authenticate different end users:**<br>`/api/v1/authn`                                                               | 100              | 600              | 600     | 600        | 500                   |
-| **Verify a factor:**<br>`/api/v1/authn/factors/${factorIdOrFactorType}/verify` only                                     | 100              | 600              | 600     | 600        | 500                   |
-| **Get session information:**<br>`/api/v1/sessions`                                                                     | 100              | 600              | 600     | 600        | 750                   |
-| **OAuth2 requests for Custom Authorization Servers:**<br>`/oauth2/${authorizationServerId}/v1` except `/oauth2/${authorizationServerId}/v1/authorize` and public metadata endpoints (see Note below)  | 300 | 1,200 | 1,200     | 1,200       | 2,000                  |
+| **Authenticate different end users:**<br>`/api/v1/authn`<br>*Eligible for dynamic scale and workforce multiplier*                                                               | 100              | 600              | 600     | 600        | 500                   |
```

Comment:
why the italics? I built locally, and it doesn't look so hot. Can we just have all of the italicized lines be regular text? We avoid the use of italics in the docs. From the style guide:

emphasis | Don't use bold, italics, or underlining for emphasis. If something needs that much attention, consider isolating it as a standalone note or tip.

Reply:
Good point. I'll remove the italics.